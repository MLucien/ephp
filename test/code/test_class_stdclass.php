<?php

$a = new stdClass;
$a->name = "John";
$a->surname = "Smith";

$b = (array)$a;
$c = (object)$b;

var_dump($a);
var_dump($b);
var_dump($c);
