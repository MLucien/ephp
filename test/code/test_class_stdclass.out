object(stdClass)#1 (2) {
  ["name"]=>
  string(4) "John"
  ["surname"]=>
  string(5) "Smith"
}
array(2) {
  ["name"]=>
  string(4) "John"
  ["surname"]=>
  string(5) "Smith"
}
object(stdClass)#2 (2) {
  ["name"]=>
  string(4) "John"
  ["surname"]=>
  string(5) "Smith"
}
